lib/beaker/shared/subcommands_util.rb

```diff
@@ -1,22 +1,30 @@
+require 'rake'
+
 module Beaker
   module Shared
     # Methods used in execution of Subcommands
-    # - should we write the config?
-    # - reset ARGV 
+    # - should we execute a subcommand?
+    # - reset ARGV
     # - execute Beaker
+    # - update a rakefile to require beaker quickstart tasks
+    # - execute a rake task
+    # - execute the vagrant quickstart task
+    # - execute the vmpooler quickstart task
+    # - exit with a specific message
+    # - verify that the init subcommand has the correct arguments
+    # - execute the quick start task for the specified hypervisor
     module SubcommandsUtil
-      CONFIG_PATH = ".beaker/config"
-    
-      @@write_config = false
+      RAKE_FILES = ["Rakefile", "rakefile", "Rakefile.rb", "rakefile.rb"]
```

Comment:
@ferglor have you looked into using rake's code itself to find these files for you, rather than reverse-engineering their rake file finding?